aws/files/autoeips.py

```diff
@@ -74,10 +120,21 @@ def associate_eip(self, eips, retries=3):
 
                 success = eip.associate(
                     instance_id=self.instance_id, allow_reassociation=False)
+                self.update_standby_mode(False)
                 return success
+        logger.warning("Failed to associate with any EIP's")
+        self.update_standby_mode(True)
         return False
 
     def get_unassociated_eips(self):
+        """
```

Comment:
I've also noticed that, when testing, if I set filter_addresses to an empty list, then the code will grab any unassociated EIP. I think this is a bit dangerous if it happens by default. 

We probably just need something like: 
```python
if not self.filter_address:
     logger.critical("EIPs must be explicitly set in the pillar.")
     self.safe_exit(1)